distribution/lib/Standard/Test/0.0.0-dev/src/Test_Environment.enso

```diff
@@ -12,4 +12,11 @@ polyglot java import org.enso.base.Environment_Utils
    `System.getenv` Java call remains unchanged.
 unsafe_with_environment_override : Text -> Text -> Any -> Any
 unsafe_with_environment_override key value ~action =
-    Environment_Utils.with_environment_variable_override key value (_->action)
+    ## This has to be done in Enso, not in Java, due to the bug: https://github.com/enso-org/enso/issues/7117
```

Comment:
I don't understand why should I create this separate file? What would be the advantage over keeping the code here?

Reply:
I removed the Java counterpart `with_environment_variable_override` as due to #7117 it is problematic.

But keeping this logic here seems to make most sense - the override is meant only for testing - it should never be used in actual production code.

Reply:
Because the original implementation was in `Environment_Utils.java`, as if it was possibly meant to be used more than once. If not, then it doesn't matter.

Reply:
I see. No that utils file was written specifically for this test tool and it shouldn't really be used elsewhere.